src/Microsoft.Unity.Analyzers.Tests/ProtectedUnityMessageTests.cs

```diff
@@ -0,0 +1,159 @@
+/*--------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See LICENSE in the project root for license information.
+ *-------------------------------------------------------------------------------------------*/
+
+using System.Threading.Tasks;
+using Xunit;
+
+namespace Microsoft.Unity.Analyzers.Tests
+{
+	public class ProtectedUnityMessageTests : BaseCodeFixVerifierTest<ProtectedUnityMessageAnalyzer, ProtectedUnityMessageCodeFix>
+	{
+		[Fact]
+		public async Task FixPrivateUnityMessage()
+		{
+			const string test = @"
+using UnityEngine;
+
+class Camera : MonoBehaviour
+{
+    private void Start()
+	{
```

Comment:
I think there is a mix of tabs/spaces in this string. For all test strings, I think it is better to use spaces given we check locations.